examples/research_projects/wav2vec2/run_alignment.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+python alignment.py  \
+--model_name="arijitx/wav2vec2-xls-r-300m-bengali" \
+--wav_dir="./wavs" \
```

Comment:
you mean a small sample dataset with folder with audio files:
```
wavs/
   audio_filename_0.wav
   audio_filename_1.wav
   audio_filename_2.wav
   ...
```
and `scripts.txt` file like:
```
audio_filename_0.wav       transcription of first file
audio_filename_1.wav       transcription of second file    
audio_filename_3.wav       transcription of third file
```
?
I don't have any on the Hub but locally I have a sample from MLLibrispeech for Polish with 127 audio files. I can upload it to the Hub but as far as I understand you need data for Bengali? :)

Reply:
> > Hey @arijitx! I took the liberty of updating your branch with some style fixes, hope you don't mind :) Now it should pass all of the checks. Also, could you upload the example files (transcriptions and speech) to a repository on HF Hub, so that the example is more reproducible?
> > As a suggestion: it would probably be tidier if we moved the example to `research_projects/wav2vec2/alignment/` @patrickvonplaten wdyt?
> 
> Thanks @anton-l for fixing the style :) @patrickvonplaten don't have much idea about how to setup a live example, can you please point me to any documentation if there is any ? Thanks in advance :)

I have the same question here @lhoestq @polinaeterna - do we have docs on how to create a short sample audio dataset ?